src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -106,6 +106,22 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
             vscode.workspace.applyEdit(edit);
           }
           break;
+        case 'getPathByDialog':
+          const dialogOptions = {
+            canSelectMany: false,
+            canSelectFolders: e.isFolder,
+            openLabel: 'Open',
+            filters: {'target files': e.ext, 'all files': ['*']}
```

Comment:
(Optional) How about this kind of approach?

```suggestion
            filters: {`target files(*.${e.ext})`: e.ext, 'all files': ['*']}
```

![image](https://user-images.githubusercontent.com/50489513/181168113-12e5035e-f6c1-4570-a255-32efef2e2149.png)

I followed how windows file open dialog shows.

Reply:
It looks good but key value should be constant I think :(
I will handle it later...!